BaseFilter/sections/css_extended.txt

```diff
@@ -6,6 +6,8 @@
 ! Bad: example.org##main > div.ad (should be in specific.txt)
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/129860
+ingles.com,spanishdict.com#?#body > div.ReactModalPortal:has(> .ReactModal__Overlay > .ReactModal__Content > div[class] > div[class] > div[class] > div[class] > iframe)
```

Comment:
I'd specify this `iframe` as the rule below
```
! https://github.com/AdguardTeam/AdguardFilters/issues/129860
ingles.com,spanishdict.com#?#body > div.ReactModalPortal:has(> div.ReactModal__Overlay > div.ReactModal__Content > div[class] iframe[title^="video"][srcdoc*="birthdayDog.mp4"])
```

Reply:
There's more than one type of iframe